docs/apppromote.yaml

```
@@ -0,0 +1,52 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: apppromote
+spec:
+  resources:
+    inputs:
+    - name: git-source
+      type: git
+  params:
+  - name: github-secret
+    type: string
+    description: seret name of the github that has the access token. The key name is token.
```

```suggestion
    description: name of the secret that contains the GitHub access token, the access token must be in a token key.
```

There are a number of places where *secret* is misspelled as *seret*.

---

I fix it.